src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFState.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.broadinstitute.hellbender.utils.mcmc.AbstractParameterizedState;
+import org.broadinstitute.hellbender.utils.mcmc.Parameter;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static java.lang.Math.min;
+
+/**
```

Comment:
might be overkill to fully document this class, but perhaps a pointer to ACSMethods.pdf (or to documentation in other classes) could be helpful.